src/include/migraphx/op/pooling.hpp

```diff
@@ -331,7 +331,7 @@ struct pooling
                 int end;
                 std::size_t dilated_kernel_dim = dilate_dim(kernel_dims[d_2], dilations[d_2]);
                 // NOLINT
-                if(count_include_pad and (mode != pooling_mode::max))
+                if(count_include_pad and mode == pooling_mode::average)
```

Comment:
Why is this change needed? The `count_include_pad` flag should only be skipped for max since it doesn't use an average of the pixel values.

Reply:
Per Onnx specfications, count_include_pad as an operator parameter is used only with pooling_mode::average. Not with other pooling modes, e.g. max or lpnorm.

Reply:
This is not an onnx operator. We have this attribute on this operator and it should work for all pooling modes that use a count.

Reply:
Ah wait, lpnorm doesnt use the count. Nevermind.